src/features/DrugChart/DrugChartWrapper/DrugChartWrapper.spec.jsx

```diff
@@ -0,0 +1,123 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import DrugChartWrapper, { TransformDrugChartData } from "./DrugChartWrapper";
+
+const drugChartData = [
+  {
+    id: 1,
```

Comment:
Same comment on moving test constants to separate file to make test files cleaner